includes/class-newspack-popups-inserter.php

```diff
@@ -219,12 +219,24 @@ public static function insert_popups_in_post_content( $content, $popups ) {
 
 		$parsed_blocks = self::convert_classic_blocks( parse_blocks( $content ) );
 
-		$parsed_blocks = array_values( // array_values will reindex the array.
+		// List of blocks that require innerHTML to render content.
+		$innerhtml_blocks_names = [
```

Comment:
The comment and variable name are a bit unclear to me. This is effectively a list of blocks which render nothing if their `innerHTML` is empty. While I don't have a great idea for the variable name, I think the comment does not reflect the intention here.

Reply:
I agree! What do you think of 93785c75ff2a9f3f9ea6be68dabb823df51d1016 ?